_posts/2016-03-09-public-speaking-part1-is-it-for-me.markdown

```diff
@@ -0,0 +1,53 @@
+---
+
+At Artsy, sharing knowledge is heavily encouraged and so we write our code in the open, document using blog posts, and speak about our experiences. Many have [bumped into us][tw_ldn] at various places around the world. And so I often get the question: “How did you get started? I’m interested in speaking too”.
+
+In the name of sharing experiences with others, I can’t encourage this enough. Unfortunately though, public speaking is still considered [man’s biggest fear][atc_fear].
+
+I’d like to share with you the benefits of public speaking, tips on how to get started, and how to give great presentations.
+
+<!-- more -->
+
+## Why do it?
+
+### Communication
+
+Without necessarily realising, you’re publicly speaking all the time. At the heart of it, it’s the act of communicating and expressing your ideas and viewpoints for others to understand.
+
+It could be as simple as ordering your morning coffee or as important as trying to convince your team that they should use a new technology. Regardless of the context, it involves restructuring your thoughts into words and metaphors that you think others can relate to in order for them to understand your concepts.
+
+**Public speaking is a skill that is always good to improve on**. Misunderstandings happen all the time, and at the heart of that is often poor communication.
+
+I know people have refrained from speaking up in meetings at work, because they struggle with speaking in front of others. There’s no need for everyone to be a keynote speaker at a conference, but if it’s hindering you in your career, it may be worth practicing. Everyone should feel empowered to call a meeting, participate in discussions, or ask questions at an all hands.
+
+### Learning
+
+Another reason giving talks is 👍🏽 is that it will allow you to learn more fully. One of the best ways to learn is by [teaching others][atc_teach]. You grasp ideas better yourself and in order to communicate them properly, you get into the habit of restructuring your ideas and concepts.
+
+Unfortunately, people often also consider this the blocker to public speaking. Not only are you afraid you’ll accidentally make a fool of yourself (“what if I trip?”) but also that it will be apparent that you don’t know something well enough. The only way to get over this is by doing it.
+
+Chances are that you are actually very knowledgeable in certain fields, you just perceive that as normal. “Yeah, but that’s not interesting… everyone knows about _that_”. And I can bet that, well, probably not everyone knows about that. Definitely not as much as you might actually do.
+
+We all have our strengths. This is the time to play on them. We can’t be experts in both Core Image, Core Animation, and the Objective C runtime, all the while knowing everything about Swift Protocols. Pick your slice from a much wider scope. It doesn’t need to be a full framework, even just a specific use case of it, like [background transfer services][talk_gwen].
```

Comment:
Agreed! There's a lot a developer can use, but we can't be experts in all of that :). WWDC is a great example.